### PR TITLE
Login Page Completed

### PR DESCRIPTION
**Pull request by priyarajputexc**

### CODE REVIEW

File: src/app/components/login/login.component.ts [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+  get email() {
+    return this.loginForm.get('email');
+  }
+  get password() {
+    return this.loginForm.get('password');
+  }
+
+  async onSubmit(formData) {
+    console.log(formData);
```
Comment (upexcel) [inline src/app/components/login/login.component.ts]: remove consoles.

Comment (priyarajputexc) [reply src/app/components/login/login.component.ts]: ok will remove them

File: src/app/components/login/login.component.html [hunk 2]
```diff
@@ -0,0 +1,33 @@
+<div class="loginCss">
+
+  <form [formGroup]="loginForm" (ngSubmit)="onSubmit(loginForm.value)">
+    <div class="form-group">
+      <label for="exampleInputEmail1">Email address</label>
+      <input type="email" class="form-control" placeholder="Enter email" id="exampleInputEmail1" formControlName="email"
```
Comment (upexcel) [inline src/app/components/login/login.component.html]: change the id name to specific to the field.

File: src/app/components/login/login.component.css [hunk 3]
```diff
@@ -0,0 +1,9 @@
+.loginCss {
```
Comment (upexcel) [inline src/app/components/login/login.component.css]: we don't use camelCase here.
it's not wrong bt we should go by the convention. i.e. login-css

**Pull request merged**